policies/antitrust-policy.md

```diff
@@ -0,0 +1,54 @@
+# Commonhaus Foundation Anti-Trust Policy
+
+## Purpose
+
+The Commonhaus Foundation (CF) requires that all members, activities, projects, committees, and working groups comply with applicable anti-trust laws.
+
+## Scope
+
+This policy applies to all participants in Commonhaus Foundation activities, including but not limited to general members, project representatives, councilors, and committee members. It is designed to complement the [Conflict of Interest Policy](./conflict-of-interest.md) and other foundation governance documents.
+
+## Commitment to Compliance
+
+By taking part in activities related to foundation governance or operations, participants agree to follow this policy and promote fair competition. This includes activities initiated on behalf of the Council, EGC, or other officially recognized committee or office.
+
+## General Principles
+
+All [participants](#scope) in foundation activities must comply with all applicable anti-trust and competition laws.
+
+Do not engage in practices that limit competition, like price-fixing, dividing markets, or bid-rigging.
+
+Avoid discussing or agreeing on topics that could seem anti-competitive, such as:
+
+- Pricing strategies or policies
+- Dividing markets or customers
+- Boycotting vendors, customers, or products
+- Limiting production or supply
+
+## Participation in Foundation activities
+
+The Foundation encourages open participation in all projects and activities.
+Membership should not be used to exclude or disadvantage anyone.
+
+If a member suspects that any activity may violate antitrust laws, they should [report their concern](#reporting-a-violation) immediately.
+If the violation happens during a meeting, ask to have your concern recorded in the minutes.
+
+All suspected violations will be promptly investigated by the CF Council.
+
+### Reporting a Violation
+
+Send an email to the [`report` mailing list](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml) describing the situation and outlining your concern.
+
+## Consequences of Violations
+
+Antitrust law violations can lead to severe consequences, including heavy fines for both companies and individuals, imprisonment in some jurisdictions, civil claims for damages by affected parties, and reputational damage to members and the CF.
+
+Violations of this policy may result in disciplinary action, including suspension or revocation of membership.
+
+## Review and Updates
+
+This policy will be reviewed periodically to ensure it remains effective and relevant. Amendments will be made in accordance with the CF’s governance processes.
+
+## Acknowledgment
+
+All members are required to acknowledge their understanding and acceptance of this policy upon joining CF and at regular intervals as determined by the CF Council.
```

Comment:
I suggest adding this acknowledgment to the onboarding process for the CF.

Reply:
Once it has been approved, I'll add it to the list of attestations.